content/influxdb/cloud/account-management/limits.md

```diff
@@ -0,0 +1,122 @@
+---
+title: InfluxDB Cloud limits
+description: >
+  InfluxDB Cloud has adjustable account limits and hard system limits.
+weight: 110
+menu:
+  influxdb_cloud:
+    parent: Account management
+    name: Limits
+products: [cloud]
+---
+
+InfluxDB Cloud has adjustable account limits per organization and hard global system limits. Review the following InfluxDB Cloud limits and errors to plan for your bandwidth needs:
+
+- [Account limits](#account-limits)
+- [Global limits](#global-limits)
+- [Limit errors](#limit-errors)
+
+<!--To estimate your projected usage costs, use the [InfluxDB Cloud pricing calculator](/influxdb/cloud/account-management/pricing-calculator/). -->
+
+## Account limits
+
+InfluxDB Cloud has adjustable limits applied per-account to reduce the chance of unexpected charges and protect the service for all users.
+
+_To request higher limits, reach out to [InfluxData Support](https://support.influxdata.com/)._
+
+### Rate limits
+
+{{% warn %}}
+Rate limits are accrued against a five-minute window. Rate limits for writes are applied to uncompressed payloads.
+{{% /warn %}}
+<!-- Include something about how the rate limit is calculated. -->
+
+#### Free Plan rate limits
+
+- Read: 300 MB data every 5 minutes
```

Comment:
We need to define what "read" means here. I have an action to confirm my understanding. Will update his PR with that once confirmed by engineering.

Reply:
Thank you, @rbetts--will stay tuned for your understanding on read.

Relatedly, cc @mavarius @gunnaraasen, I notice during Cloud signup, we say that limits for Free plan are:
- 1000 kb/s reads 
- 17 kb/s writes

per [conversation](https://github.com/influxdata/DAR/issues/224#issuecomment-906717547) with Phil, do you think best to keep the MB/5 min measure? Seems  per sec maybe less explicit and "average over 5 minutes" may be a bit awkward?

Reply:
@rbetts were you able to confirm the exact definition of read? This is the most recent version of what's in this PR:

> - **Read**: 300 MB per 5 minutes (response bytes)

I think we should specify whether it's the bytes of the full HTTP response or just the bytes of the response payload (i.e. the query result size).